tests/unit/IonBinaryReaderTest.js

```diff
@@ -30,6 +30,12 @@ define([
                 reader.next();
                 assert.deepEqual(reader.timestampValue(), timestamp);
             },
+
+            'deepValue': () => {
+              var ivm = [0xe0, 0x01, 0x00, 0xea];
+              var ionReader = ion.makeReader(ivm);
+              assert.deepEqual(ionReader.deepValue(), new Uint8Array(ivm));
```

Comment:
Oh that is interesting, I would not expect `deepValue()` to work on a `Reader` that is not yet positioned on a value.  The IVM is not a value, per se, so I think we need to be a bit careful here.  E.g. what is the implication of doing this when there are local symbol table(s).